yb-voyager/src/tgtdb/event.go

```diff
@@ -361,6 +361,10 @@ func (eb *EventBatch) GetChannelMetadataUpdateQuery(migrationUUID uuid.UUID) str
 		migrationUUID, eb.ChanNo)
 }
 
+func (eb *EventBatch) GetEventID() string {
```

Comment:
Nit: name this method as just `ID()`. So the call would look like `eventBatch.ID()`.